source/breakfast/index.rst

```diff
@@ -4,4 +4,4 @@ Breakfast
 .. toctree::
    :maxdepth: 1
 
-   pancakes
+   jennaPancakes
```

Comment:
This is missing `bernardoPancakes` I think?

Reply:
Fixed.